<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometryOptionsDialog</class>
 <widget class="QDialog" name="GeometryOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geometry Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QPushButton" name="mCancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="mOkButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
       <widget class="QPushButton" name="mRevertOptionsButton">
         <property name="text">
           <string>Revert to Default</string>
         </property>
       </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="mOptionsListWidget">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Focus Point Movement Options</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Show/Hide Options</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="mStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mFocusPointOptionsPage">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="0">
        <widget class="QPushButton" name="mMoveFPLeftButton">
         <property name="text">
          <string>Move Focus Point Left [Left]</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="mMoveFPResetButton">
         <property name="text">
          <string>Reset Focus Point [R]</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="mMoveFPDownButton">
         <property name="text">
          <string>Move Focus Point Aft [Down]</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="mMoveFPUpButton">
         <property name="text">
          <string>Move Focus Point Forward [Up]</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="mMoveFPRightButton">
         <property name="text">
          <string>Move Focus Point Right [Right]</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mShowHideOptionsPage">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="2" column="0">
        <widget class="QCheckBox" name="mShowEnginesCB">
         <property name="text">
          <string>Show Engines [E]</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="mShowPointMassesCB">
         <property name="text">
          <string>Show Point Masses [M]</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="mShowSelectedCGCB">
         <property name="text">
          <string>Show Selected Object CG Location [S]</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="mShowPrimaryAxesCB">
         <property name="text">
          <string>Show Primary Axes [X]</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="mShowFuelTanksCB">
         <property name="text">
          <string>Show Fuel Tanks [F]</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QGroupBox" name="mShowAeroVectorsGroupBox">
         <property name="title">
          <string>Aero Vectors</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QRadioButton" name="mNoAeroVectorsButton">
            <property name="text">
             <string>Hide Aero Vectors [H]</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="mSelectedObjectOnlyButton">
            <property name="text">
             <string>Selected Object Only [V]</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="mEntireVehicleButton">
            <property name="text">
             <string>Show All Aero Vectors [A]</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="mShowWireframeCB">
            <property name="text">
             <string>Show Wireframe</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="mShowLandingGearCB">
         <property name="text">
          <string>Show Landing Gear [G]</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="mShowSpeedBrakesCB">
         <property name="text">
          <string>Show Speed Brakes [B]</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="mShowVehicleCGCB">
         <property name="text">
          <string>Show Vehicle CG [C]</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="mShowThrustVectorsCB">
         <property name="text">
          <string>Show Thrust Vectors [T]</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="mShowDebugCB">
         <property name="text">
          <string>Show Debug Data [F10]</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
         <widget class="QCheckBox" name="mUseDefaultsCB">
           <property name="text">
             <string>Use Defaults on Startup</string>
           </property>
         </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
